kettle-plugins/mapreduce/pom.xml

```diff
@@ -72,6 +72,11 @@
       <version>${pdi.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.dom4j</groupId>
+      <artifactId>dom4j</artifactId>
+      <scope>provided</scope>
```

Comment:
@pentaho-lmartins I need to check the reasoning for the provided with you.

Reply:
The `dom4j` is included in the assembly thus the `provided` scope.